planner/core/exhaust_physical_plans.go

```diff
@@ -815,8 +814,10 @@ func (la *LogicalAggregation) getStreamAggs(prop *property.PhysicalProperty) []P
 		// The table read of "CopDoubleReadTaskType" can't promises the sort
 		// property that the stream aggregation required, no need to consider.
 		for _, taskTp := range []property.TaskType{property.CopSingleReadTaskType, property.RootTaskType} {
+			//Add taskTp in childrenReqProps, it will pass to it's child in findBestTask
```

Comment:
Please add space between `//` and first word, and add `.` in the end of the sentence.